host_guest/GDCC_and_guests/README.md

```diff
@@ -35,5 +35,4 @@ For positively charged guests, chloride salts are expected to be used.
 # What's here
 
 All files here so far are extremely tentative pending finalization.
-- Hosts: `OA.pdb` and `exoOA.pdb` provided by Paolo Suating (Tulane) April 26, 2019; `SAMPL7_tentative_hosts.cdx` provided by Bruce Gibb April 26, 2019.
-- Guests: `SAMPL7_tentative_guests.cdx` provided by Bruce Gibb April 26, 2019.
+- `hosts_guest_files/`: Directory containing Chemdraw (`.cdx`), PDB, MOL2 and SDF files.
```

Comment:
Why did you put these under another subdirectory? I suppose could be OK, but then leaves the higher-level directory nearly empty.

Also, why `hosts_guest_files` as if there are many hosts and one guest? Probably I'd call it `host_guest_files`.

Reply:
I added the cdx files up a directory, but I think the host PDBs should stay in the subdirectory because that's where the guest PDBs are stored. Will rename the folder

Reply:
I guess I was asking, why not have all the files in the base directory for `GDCC_and_guests`?

Either way, you need to end up with consistent organization across all three host-guest systems.